src/components/portrait/portrait.d.ts

```diff
@@ -0,0 +1,52 @@
+import * as React from "react";
+
+export type PortraitShapes = "circle" | "square";
+
+export type PortraitSizes = "XS" | "S" | "M" | "ML" | "L" | "XL" | "XXL";
+
+export interface PortraitBaseProps {
+  /** The size of the Portrait. */
+  size?: PortraitSizes;
+  /** The `alt` HTML string. */
+  alt?: string;
+  /** The shape of the Portrait. */
+  shape?: PortraitShapes;
+  /** The initials to render in the Portrait. */
+  initials?: string;
+  /** Use a dark background. */
+  darkBackground?: boolean;
+  /** Prop for `onClick` events. */
+  onClick?: React.MouseEvent;
```

Comment:
I believe this type should actually be `onClick?: (ev: React.MouseEvent<HTMLElement>) => void;`